src/wallet/rpcwallet.cpp

```diff
@@ -757,7 +757,7 @@ UniValue movecmd(const JSONRPCRequest& request)
             "1. \"fromaccount\"   (string, required) The name of the account to move funds from. May be the default account using \"\".\n"
             "2. \"toaccount\"     (string, required) The name of the account to move funds to. May be the default account using \"\".\n"
             "3. amount            (numeric) Quantity of " + CURRENCY_UNIT + " to move between accounts.\n"
-            "4. minconf           (numeric, optional, default=1) Only use funds with at least this many confirmations.\n"
+            "4. minconf           (numeric, optional) Ignored. Remains for backward compatibility.\n"
```

Comment:
µnit: Might want to change the name into something meaningless as well? Like `4. num     (numeric, optional)`